macros/cross-adapter-modeling/base/segment_web_page_views.sql

```diff
@@ -32,9 +32,8 @@ renamed as (
         search as page_url_query,
         
         referrer,
-        ltrim({{ dbt_utils.get_url_host('referrer') }}, 'www.')::varchar 
-            as referrer_host,
-        
+        ltrim({{ dbt_utils.safe_cast(dbt_utils.get_url_host('referrer'), 'string') }}, 'www.') as referrer_host,
```

Comment:
I'm moving the cast inside so the macro call is cleaner. If there's a reason why this cast should occur outside of `ltrim`, let me know. I'm not sure why this cast is happening in the first place, since `LTRIM` only accepts string expressions to begin with. Should we just drop the cast?

Reply:
Yep, we can replace this with:
```suggestion
        ltrim({{ dbt_utils.get_url_host('referrer') }}, 'www.') as referrer_host,
```
The [get_url_host macro](https://github.com/fishtown-analytics/dbt-utils/blob/master/macros/web/get_url_host.sql) actually does the casting for us!

Reply:
I wondered about that! Awesome.